Code/Editor/Core/LevelEditorMenuHandler.cpp

```diff
@@ -758,24 +758,24 @@ QMenu* LevelEditorMenuHandler::CreateHelpMenu()
     layout->addWidget(lineEdit);
     containerWidget->setLayout(layout);
     containerWidget->setContentsMargins(2, 0, 2, 0);
-    lineEdit->setPlaceholderText(tr("Search documentation"));
+    lineEdit->setPlaceholderText(tr("Search documentation..."));
     lineEditSearchAction->setDefaultWidget(containerWidget);
 
     auto searchAction = [lineEdit]()
         {
             auto text = lineEdit->text();
             if (text.isEmpty())
             {
-                QDesktopServices::openUrl(QUrl("https://o3de.org/docs/"));
+                QDesktopServices::openUrl(QUrl("https://www.o3de.org/docs/"));
             }
             else
             {
                 static const int versionStringSize = 128;
                 char productVersionString[versionStringSize];
                 const SFileVersion& productVersion = gEnv->pSystem->GetProductVersion();
                 productVersion.ToString(productVersionString, versionStringSize);
-
-                QUrl docSearchUrl("https://o3de.org/docs/");
+                
```

Comment:
Looks like some tab space got inserted here accidentally?

Reply:
Looks fine on my machine, not sure why the diff shows it like this :D

Reply:
Ok the new submission fixed the diff :)